automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -209,6 +235,192 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLike w
+            val submissionPatience = PatienceConfig(timeout = scaled(Span(8, Minutes)), interval = scaled(Span(30, Seconds)))
+            implicit val patienceConfig: PatienceConfig = submissionPatience
+
+            // Get workflow ID from submission details
+            val rootWorkflowId: String = eventually {
+              val (status, workflows) = Rawls.submissions.getSubmissionStatus(projectName, workspaceName, submissionId)
+              withClue(s"queue status: ${getQueueStatus()}, submission status: ${getSubmissionResponse(projectName, workspaceName, submissionId)}") {
+                status should (be("Submitted") or be("Done")) // very unlikely it's already done, but let's handle that case.
+                workflows should not be (empty)
+                workflows.head
+              }
+            }
+
+            // Get sub-workflow ids and check the `zones` in workflow options belong to `europe-north1` region
+            val subWorkflowIds: List[String] = eventually {
+              val rootWorkflowMetadata = Rawls.submissions.getWorkflowMetadata(projectName, workspaceName, submissionId, rootWorkflowId)
+              val workflowOptions = parseWorkflowOptionsFromMetadata(rootWorkflowMetadata)
+              val subWorkflowIds = parseSubWorkflowIdsFromMetadata(rootWorkflowMetadata)
+
+              withClue(getWorkflowResponse(projectName, workspaceName, submissionId, rootWorkflowId)) {
+                workflowOptions should include (europeNorth1ZonesPrefix)
+
+                subWorkflowIds should not be (empty)
+                subWorkflowIds.length shouldBe(3)
+
+                subWorkflowIds
+              }
+            }
+
+            // once the sub-workflows have finished running, check
+            //    - the zones for each job that were determined by Cromwell and
+            //    - the worker assigned for the tasks
+            // belong to `europe-north1`
+            eventually {
+              val subWorkflowsMetadata = subWorkflowIds map { Rawls.submissions.getWorkflowMetadata(projectName, workspaceName, submissionId, _) }
+              val subWorkflowCallMetadata = subWorkflowsMetadata flatMap parseCallsFromMetadata
+              val callZones = parseRuntimeAttributeKeyFromMetadata(subWorkflowCallMetadata, "zones")
+              val workerAssignedExecEvents = parseWorkerAssignedExecEvents(subWorkflowCallMetadata)
+
+              subWorkflowsMetadata foreach { x => x should include (""""executionStatus":"Done"""") }
+
+              callZones foreach { _.split(",") foreach { zone => zone should startWith (europeNorth1ZonesPrefix) } }
+
+              workerAssignedExecEvents should not be (empty)
+              workerAssignedExecEvents foreach { event => event should include (europeNorth1ZonesPrefix) }
+            }
+          }
+        }
+      }
+    }
+
+    "should be able to run sub-workflow tasks in a cloned workspace in non-US regions" in {
```

Comment:
If the "cloned workspace" test works, do we still need the "before we cloned it workspace" version of the test?

Reply:
The two tests are different because one runs a workflow in a newly-created workspace, while the other runs a workflow in a cloned workspace.

Reply:
My thought was just that if you're able to "create and clone" a workspace in a different region, and everything works correctly, does that automatically imply that the "just create" case must also work? (and therefore that we don't need to test it separately).

I think you're definitely being more thorough, fwiw, I was just wondering whether we could economize if these tests turn out to be extremely slow.

Reply:
I would still prefer to be more thorough, at least for now as this is a new feature. We can always come back and remove a test if we realize it's being slow or not needed anymore